<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userpage</class>
 <widget class="QWidget" name="userpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1301</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Homepage
{

background-color: rgb(222,220,220)
}

QTextEdit, QPlainTextEdit
{
	border : 1px solid white;
	border-radius: 2px;
}
QPushButton
{
	color: black;
	background: rgb(255,255,255);
	border: 2px solid white;
	border-radius: 2px;
	text-align : left;
}
</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>438</x>
     <y>-10</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:700; font-style:italic; color:#aa330b;&quot;&gt;RENT&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>568</x>
     <y>-10</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#textEdit_2
{
	border-color: white;
	bordrer: none;
}</string>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:700; font-style:italic; color:#00007f;&quot;&gt;US&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-4</y>
     <width>1311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#plaintextEdit
{
	border: 2px solid brown;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>6</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_3
{
	color: black;
	background: rgb(255,255,255);
	border: 2px solid white;
	border-radius: 2px;
}
#pushButton_3:hover
{
	background: rgb(220,220,200);
}</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>-15</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_5
{
	color: black;
	background: rgb(255,255,255);
	border: 0px;
	border-radius: 0px;
	pading: 2px;
}
#pushButton_5:hover
{
	background: rgb(20,20,20);
	color: white;
}
#pushButton_5::menu-indicator{ image: none; }</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>6</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_4
{
color: black;
	background: rgb(255,255,255);
	border: 2px solid white;
	border-radius: 2px;
}
#pushButton_4:hover
{
	background: rgb(220,220,200);
}</string>
   </property>
   <property name="text">
    <string>About Us</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_3">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>10</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; font-style:italic;&quot;&gt;Meet up for the good&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_4">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>2</y>
     <width>104</width>
     <height>51</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#textEdit_4
{
	border-image: url(:/image/images/user.png);
}</string>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>201</y>
     <width>187</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <bold>true</bold>
        <stylestrategy>PreferDefault</stylestrategy>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton
{
color: black;
	background: rgb(255,255,255);
	border: 2px solid white;
	border-radius: 2px;
}
#pushButton:hover
{
	background: rgb(220,220,200);
}</string>
      </property>
      <property name="text">
       <string>  Dashboard</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/dashboard.png</normaloff>images/dashboard.png</iconset>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <bold>true</bold>
        <stylestrategy>PreferDefault</stylestrategy>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_2
{
color: black;
	background: rgb(255,255,255);
	border: 2px solid white;
	border-radius: 2px;
}
#pushButton_2:hover
{
	background: rgb(220,220,200);
}</string>
      </property>
      <property name="text">
       <string>  My Contribution</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/report.png</normaloff>images/report.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <bold>true</bold>
        <stylestrategy>PreferDefault</stylestrategy>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_6
{
color: black;
	background: rgb(255,255,255);
	border: 2px solid white;
	border-radius: 2px;
}
#pushButton_6:hover
{
	background: rgb(220,220,200);
}</string>
      </property>
      <property name="text">
       <string>  Add a Property</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/home.png</normaloff>images/home.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_7">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <bold>true</bold>
        <stylestrategy>PreferDefault</stylestrategy>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_7
{
color: black;
	background: rgb(255,255,255);
	border: 2px solid white;
	border-radius: 2px;
}
#pushButton_7:hover
{
	background: rgb(220,220,200);
}</string>
      </property>
      <property name="text">
       <string>  About Me</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/user.png</normaloff>images/user.png</iconset>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>plainTextEdit</zorder>
  <zorder>commandLinkButton</zorder>
  <zorder>textEdit</zorder>
  <zorder>textEdit_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>textEdit_3</zorder>
  <zorder>textEdit_4</zorder>
  <zorder>commandLinkButton_2</zorder>
  <zorder>commandLinkButton_3</zorder>
  <zorder>commandLinkButton_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
